src/Mvc/Mvc.Analyzers/test/Mvc.Analyzers.Test.csproj

```diff
@@ -16,6 +16,7 @@
   <ItemGroup>
     <None Include="xunit.runner.json" CopyToOutputDirectory="PreserveNewest" />
     <Compile Include="$(SharedSourceRoot)test\SkipOnHelixAttribute.cs" />
+    <Compile Remove="TestFiles\TagHelpersInCodeBlocksAnalyzerTest\*.*" />
```

Comment:
?

Reply:
The C# code in the file is invalid, without this the project attempts to compile the test files which are purposely non-compileable.